### DISCUSSION TITLE
MAUI app .csproj `ApplicationDisplayVersion` and `ApplicationVersion`: Purpose?

### DISCUSSION THREAD

**Discussion by ITaluone** [dotnet/maui#12992] [Q&A] [answered]
Hi

I have a quick question: What purpose do have these two csproj parameters?

If I set a version here, the package's version is _always_ `1.0.0.0` after publish.

If I omit these two, the package's version is `1.0.0.0` as well. So: why do I need this (apparently unnecessary) versions?

FWIW: The one and only way to version a MAUI app is to edit the `Package.appxmanifest`.

**Comment by baaaaif** [comment, marked as answer]
> Hi
> 
> I have a quick question: What purpose do have these two csproj parameters?
> 
> If I set a version here, the package's version is _always_ `1.0.0.0` after publish.
> 
> If I omit these two, the package's version is `1.0.0.0` as well. So: why do I need this (apparently unnecessary) versions?
> 
> FWIW: The one and only way to version a MAUI app is to edit the `Package.appxmanifest`.

take a look at 
[https://github.com/dotnet/maui/blob/main/src/SingleProject/Resizetizer/src/GeneratePackageAppxManifest.cs#L107](https://github.com/dotnet/maui/blob/main/src/SingleProject/Resizetizer/src/GeneratePackageAppxManifest.cs#L107)

if you set your version in Package.appxmanifest to "0.0.0.0" it will take the numbers from ApplicationDisplayVersion and ApplicationVersion

**Reply by ITaluone** [reply]
Ahh.. Thanks for clarification 👍

**Reply by MartyIX** [reply]
Permalink: https://github.com/dotnet/maui/blob/83c3ac902ea496e409026c830ccd930c4021f8ab/src/SingleProject/Resizetizer/src/GeneratePackageAppxManifest.cs#L107

What is that `Resizetizer`? It's an unusual word.
